challenges/dog-photo-gallery/script.js

```diff
@@ -0,0 +1,20 @@
+document.getElementById('btn-click').addEventListener('click', () => {
+    fetch("https://dog.ceo/api/breeds/image/random").then(Response => Response.json())
```

Comment:
We tend to use lowerCamelCase for variable names in JavaScript, not UpperCamelCase - can you fix up this name?